mera/mera_pymer4.py

```diff
@@ -1,10 +1,50 @@
-from typing import List, Optional
-
+import natsort
 import numpy as np
 import pandas as pd
+from typing import List, Optional, NamedTuple
```

Comment:
Don't need `List` import (see below). Also these should be sorted properly.
```suggestion
from typing import NamedTuple, Optional
```